profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -450,11 +450,25 @@ function nexteuropa_multilingual_path_update($path) {
  */
 function _nexteuropa_multilingual_path_und_sync($path) {
 
-  // Only apply this logic to nodes that don't have
-  // a translation mechanism different from entity translation.
-  $node = menu_get_object("node", 1, $path['source']);
-  if (isset($node->type)) {
-    $translation_mechanism = variable_get('language_content_type_' . $node->type, 0);
+  // We must pass through a basic path parsing and a db_select because:
+  // - The menu_get_object function makes a access control and a redirection to
+  // the home page if users do not have access permission.
+  // (see https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-416)
+  // - The logic is only applied to nodes that have "entity translation" as
+  // translation mechanism.
```

Comment:
Should be "The logic is only applied to nodes that *don't* have "entity translation" as translation mechanism"